src/nfa/operations.cc

```diff
@@ -56,6 +56,126 @@ namespace {
         return lts_for_simulation.compute_simulation();
     }
 
+    bool vec_contains(std::vector<std::pair<State, State>> vec, std::pair<State, State> par){
```

Comment:
1) instead of passing the first argument by value, it should be better to pass by constant reference
2) anyway, using a vector here does not look like a good choice

Reply:
This function no longer exists .. (Tested the algorithm a bit after removing it ... it made the code like 3x faster)